src/index.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './pages';
-import { AuthProvider, ThemeProvider } from './providers';
+import { AuthProvider, ProductProvider, ThemeProvider } from './providers';
 import reportWebVitals from './reportWebVitals';
 import './styles/index.scss';
 
 ReactDOM.render(
   <React.StrictMode>
     <AuthProvider>
       <ThemeProvider>
```

Comment:
you can create file just with providers and import them all at one place and the here only one which imports all, it keeps index.js clean

Reply:
That would require to create an extra file just to put all providers together. Though that can be done, this is also a good approach.